src/main/java/com/conveyal/datatools/manager/jobs/NotifyUsersForSubscriptionJob.java

```diff
@@ -16,9 +17,8 @@
 /**
  * Created by landon on 6/6/16.
  */
-public class NotifyUsersForSubscriptionJob implements Runnable {
+public class NotifyUsersForSubscriptionJob extends MonitorableJob {
```

Comment:
In extending the MonitorableJob, this will now expose these notification jobs to the user as they enqueued and ran. I don't think this would be very desirable for a user to suddenly see these in the UI. Therefore, I think the previous state of this class is preferable. Either that or maybe MonitorableJob needs to be changed in a way that allows jobs to not show up in the list of user jobs.

Reply:
Yea, I hadn't fully thought through the implications of that change, but I agree that if we can handle these changes without this change, we should.

Reply:
Updated.